locales/bg-BG.json

```diff
@@ -1149,8 +1149,8 @@
       },
       "example_phrases": {
         "0": [
-          "Пътувайте за 1 спирка.",
-          "Пътувайте за 3 спирки."
+          "Travel 1 stop.",
+          "Travel 3 stops."
```

Comment:
for some reason, some `example_phrases` were reverted to the original (as it should be). however, most of them in most files are also translated..